### ISSUE TITLE
[DataGrip] Using "ColorSettings..." has several issues

### ISSUE THREAD

**Issue by devployment** [arcticicestudio/nord-jetbrains#166]
I'm using the `Color Settings...` in DataGrip, to color my databases. Mostly for production databases to be always alarmed that I'm looking at production data

![image](https://user-images.githubusercontent.com/7137845/98341402-0ecf5880-200f-11eb-9bda-cf69459fec04.png)

Nord Theme seems to have serious trouble with this. I have the following issues

The tab should be shown in light red (in my case), because the color setting for the db is "red"

![image](https://user-images.githubusercontent.com/7137845/98340911-5dc8be00-200e-11eb-94c7-a67d65d3f8b6.png)

Menu bars have "after glow" artifacts. Seems to be related  to on hover effects

![image](https://user-images.githubusercontent.com/7137845/98340999-7c2eb980-200e-11eb-9ee3-9b3cf0d2d40b.png)

"Status overlays" e.g. "Processing..." stay visible after the process finished

![image](https://user-images.githubusercontent.com/7137845/98341104-9f596900-200e-11eb-901f-3d54c477a924.png)

I'm not sure if this is all related to the Nord Theme or DataGrip itself. But other themes do not show these issues with the Color Settings enabled.

Unfortunately this makes the theme quite unusable to me in DataGrip. It's sad because I really like it in general. 

Will try tweak some settings and see if I can fine more details. Happy for any input that might make the Nord theme work in this kind of special scenario. :)

**Comment by arcticicestudio** [comment]
Hi @devployment 👋🏼, thanks for your contribution :+1:

I was not aware that there are extra color settings for such components. I guess this needs some reproduction to check which UI theme keys are used for these.
Feel free to submit a PR to improve the highlighting, otherwise I'll pick it up for the next development cycle.

**Comment by ericop** [comment]
Yeah this is still an issue, both in the "New UI" and without it turned on for DataGrip. I get all kinds of weird shadowing

![ej-2023-04-27 13_20_22-Window](https://user-images.githubusercontent.com/5218249/234941170-f4df381e-bf4a-49a7-925d-7e5a0e976c0b.png)
